pallets/asset-handler/src/lib.rs

```diff
@@ -428,6 +433,22 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Withdraw Fee
+		#[pallet::weight((195_000_000).saturating_add(T::DbWeight::get().writes(1 as Weight)))]
+		pub fn withdraw_fee(origin: OriginFor<T>, beneficiary: T::AccountId) -> DispatchResult {
+			T::AssetCreateUpdateOrigin::ensure_origin(origin)?;
+			let reducible_balance: u128 =
+				T::Currency::reducible_balance(&chainbridge::Pallet::<T>::account_id(), true)
+					.saturated_into();
+			T::Currency::transfer(
+				&chainbridge::Pallet::<T>::account_id(),
+				&beneficiary,
+				reducible_balance.saturated_into(),
+				ExistenceRequirement::KeepAlive,
+			)?;
+			Ok(())
```

Comment:
I assume this can be replaced with just:
```rust
T::Currency::transfer(
	&chainbridge::Pallet::<T>::account_id(),
	&beneficiary,
	reducible_balance.saturated_into(),
	ExistenceRequirement::KeepAlive,
)
```
since `Currency::transfer` returns `DispatchResult`.